SimPEG/utils/pgi_utils.py

```diff
@@ -164,6 +173,7 @@ def make_PGI_regularization(
     wiresmap=None,
```

Comment:
Same thing here, we are duplicating a LOT of code. Can we just keep one and be explicit about the assumption. Even if temporarily the volume part would be an issue in certain cases, we can fix the assumption after.